client/example-cook.json

```diff
@@ -0,0 +1,25 @@
+{
+    "defaults":{
+        "cluster": "dev",
+        "run": {
+            "mem": 128,
+            "cpus": 1
+        }
+    },
+    "clusters": [
+        {
+            "name": "dev",
+            "url": "http://127.0.0.1:12321/swagger-docs"
+        },
+        {
+            "name": "fake-dev0",
+            "url": "http://127.0.0.1:12322/swagger-docs",
+            "http_client": ["basic_auth", "127.0.0.1:12322", "u", "p"]
```

Comment:
I mentioned this in more detail in the basecamp post, but it's not `[auth-type, url, user, password]`, it's `[function-that-returns-an-http-client, *args]`. I can still make it a map, but it's not just for `auth`.